Vision "manual" system tests are broken

https://circleci.com/gh/GoogleCloudPlatform/google-cloud-python/2591

```
nox > Session system_tests(python_version='3.6') successful. :)
nox > Running session system_tests_manual_layer(python_version='2.7')
nox > virtualenv /var/code/gcp/.nox/system_tests_manual_layer-python_version-2-7 -p python2.7
nox > chdir /var/code/gcp/vision
nox > pip install --upgrade pytest ../core/ ../storage/
nox > pip install --upgrade ../test_utils/
nox > pip install --upgrade -e .
nox > py.test --quiet tests/system_old.py
............FFF................
=================================== FAILURES ===================================
_______________ TestVisionClientLabel.test_detect_labels_content _______________
Traceback (most recent call last):
  File "/var/code/gcp/vision/tests/system_old.py", line 368, in test_detect_labels_content
    self._assert_label(label)
  File "/var/code/gcp/vision/tests/system_old.py", line 357, in _assert_label
    self.assertIn(label.description, self.DESCRIPTIONS)
  File "/usr/local/lib/python2.7/unittest/case.py", line 803, in assertIn
    self.fail(self._formatMessage(msg, standardMsg))
  File "/usr/local/lib/python2.7/unittest/case.py", line 410, in fail
    raise self.failureException(msg)
AssertionError: u'motor vehicle' not found in ('car', 'vehicle', 'land vehicle', 'automotive design', 'wheel', 'automobile make', 'luxury vehicle', 'sports car', 'performance car', 'automotive exterior')
...
```

--- a/vision/nox.py
+++ b/vision/nox.py
@@ -35,10 +35,16 @@ def unit_tests(session, python_version):
     session.install('-e', '.')
 
     # Run py.test against the unit tests.
-    session.run('py.test', '--quiet',
-        '--cov=google.cloud.vision', '--cov=google.cloud.vision_v1',
-        '--cov-append', '--cov-config=.coveragerc', '--cov-report=',
-        'tests/',
+    session.run(
+        'py.test',
+        '--quiet',
+        '--cov=google.cloud.vision',
+        '--cov=google.cloud.vision_v1',
+        '--cov-append',
+        '--cov-config=.coveragerc',
+        '--cov-report=',
+        'tests',
+        *session.posargs
     )
 
 
@@ -63,7 +69,12 @@ def system_tests(session, python_version):
     session.install('-e', '.')
 
     # Run py.test against the unit tests.
-    session.run('py.test', '--quiet', 'tests/system.py')
+    session.run(
+        'py.test',
+        '--quiet',
+        os.path.join('tests', 'system.py'),
+        *session.posargs
+    )
 
 
 @nox.session
@@ -84,7 +95,12 @@ def system_tests_manual_layer(session, python_version):
     session.install('-e', '.')
 
     # Run py.test against the unit tests.
-    session.run('py.test', '--quiet', 'tests/system_old.py')
+    session.run(
+        'py.test',
+        '--quiet',
+        os.path.join('tests', 'system_old.py'),
+        *session.posargs
+    )
 
 
 @nox.session

--- a/vision/setup.py
+++ b/vision/setup.py
@@ -25,7 +25,7 @@ with io.open(os.path.join(PACKAGE_ROOT, 'README.rst'), 'r') as readme_file:
     readme = readme_file.read()
 
 REQUIREMENTS = [
-    'google-cloud-core >= 0.25.0, < 0.26dev',
+    'google-cloud-core >= 0.26.0, < 0.27dev',
     'google-gax >= 0.15.13, < 0.16dev',
     'googleapis-common-protos[grpc] >= 1.5.2, < 2.0dev',
 ]

--- a/vision/tests/system_old.py
+++ b/vision/tests/system_old.py
@@ -332,17 +332,23 @@ class TestVisionClientFace(BaseVisionTestCase):
 
 
 class TestVisionClientLabel(BaseVisionTestCase):
+
     DESCRIPTIONS = (
-        'car',
-        'vehicle',
-        'land vehicle',
-        'automotive design',
-        'wheel',
         'automobile make',
-        'luxury vehicle',
-        'sports car',
-        'performance car',
+        'automotive design',
         'automotive exterior',
+        'automotive wheel system',
+        'car',
+        'land vehicle',
+        'luxury vehicle',
+        'motor vehicle',
+        'muscle car',
+        'performance car',
+        'personal luxury car',
+        'rim',
+        'sports car',
+        'vehicle',
+        'wheel',
     )
 
     def setUp(self):

